scripts/zones/Port_Bastok/npcs/Clarion_Star.lua

```diff
@@ -0,0 +1,61 @@
+-----------------------------------
+-- Area: Port Bastok
+-- NPC: Clarion Star
+-- Trust NPC
+-- !pos 81.478 7.500 -24.169 236
+-----------------------------------
+local ID = require("scripts/zones/Port_Bastok/IDs")
+require("scripts/globals/keyitems")
+require("scripts/globals/npc_util")
+require("scripts/globals/quests")
+require("scripts/globals/settings")
+require("scripts/globals/trust")
+-----------------------------------
+
+function onTrade(player,npc,trade)
+    local hasPermit = player:hasKeyItem(tpz.ki.WINDURST_TRUST_PERMIT) or
+                      player:hasKeyItem(tpz.ki.BASTOK_TRUST_PERMIT) or
+                      player:hasKeyItem(tpz.ki.SAN_DORIA_TRUST_PERMIT)
+
+    if hasPermit and trade:getSlotCount() == 1 and trade:getItemSubId(0) ~= 0 and trade:getItemId(0) >= 10112 and trade:getItemId(0) <= 10193 then
+        player:setLocalVar("TradingTrustCipher", trade:getItemSubId(0))
+        player:startEvent(437, 0, 0, 0, trade:getItemId(0))
+    end
+end
+
+function onTrigger(player,npc)
+    local TrustSandoria = player:getQuestStatus(SANDORIA, tpz.quest.id.sandoria.TRUST_SANDORIA)
+    local TrustBastok   = player:getQuestStatus(BASTOK, tpz.quest.id.bastok.TRUST_BASTOK)
+    local TrustWindurst = player:getQuestStatus(WINDURST, tpz.quest.id.windurst.TRUST_WINDURST)
+    local lvl = player:getMainLvl()
+
+    if lvl >= 5 and ENABLE_TRUST_QUESTS then
```

Comment:
Now that `lvl` is only being used once (since the ifs were reformed), you can probably just grab the player's level during the `if` 😉